cmd/minikube/cmd/start_flags.go

```diff
@@ -118,6 +118,7 @@ const (
 	defaultSSHUser          = "root"
 	defaultSSHPort          = 22
 	listenAddress           = "listen-address"
+	vmwareNetworkType       = "vmware-network-type"
```

Comment:
@zarenner sorry for the changing in my opinion, but how about we called this flag "network-type"
and then then specify this is only currently used for for vmwware and then if user specificed a value for this flag for non-vmware drivers, we should Exit with Exit With Usage

```
				exit.Message(reason.Usage, "network-type flag is currently only supported only by the vm-ware driver "
```